### PR TITLE
FlushMeshSync button in actions column of environment settings centered

### LINKED ISSUE #6674: Flush MeshSync button does not appear at the center
#### Current Behavior
Flush MeshSync button in the actions column of environment settings is not at the center

before

#### Desired Behavior

after

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by deepakdinesh1123**
## Result:
after

Signed-off-by: deepakdinesh1123 <d.deepakdinesh13@gmail.com>

**Notes for Reviewers**

This PR fixes #6674 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @deepakdinesh1123

File: ui/components/MeshConfigComponent.js [hunk 1]
```diff
@@ -94,6 +95,16 @@ const styles = (theme) => ({
   },
   uploadCluster : {
     overflow : "hidden"
+  },
+  MenuItem : {
+    backgroundColor : "white",
```
Comment (Chadha93) [inline ui/components/MeshConfigComponent.js:100]:
```suggestion
    backgroundColor : theme.palette.common.white,
```

File: ui/components/MeshConfigComponent.js [hunk 2]
```diff
@@ -94,6 +95,16 @@ const styles = (theme) => ({
...
+    "&:hover" : {
+      backgroundColor : "white"
```
Comment (Chadha93) [inline ui/components/MeshConfigComponent.js:102]:
```suggestion
      backgroundColor : theme.palette.common.white
```

**Comment by Chadha93** [comment]
@deepakdinesh1123 Merge conflicts....

**Review by leecalcote — APPROVED** [review]
Looking good. 👍

**Review by Chadha93 — APPROVED** [review]
Thank you @deepakdinesh1123 LGTM

**Pull request merged**